### DISCUSSION TITLE
Advice about `group_size`

### DISCUSSION THREAD

**Discussion by goodluck-xyz** [deepmodeling/dpdispatcher#247] [General]
Can you add an parameter `group_number` ?  The `group_number` means the machine quantity that user hope finish total jobs. I know this parameter is the same as the effect of the `group_size`.  Compared to `group_size`,  `group_number`  should be controlled by the user.

**Comment by goodluck-xyz** [comment]
Such as  `mpirun -np 16 [command]`  The `group_number` is the `-np`.

**Comment by HuangJiameng** [comment]
I think you mean `group_number` = `total_task_number`/`group_size`, instead of the number of the cores we use? Maybe we can set `group_number` as an alternative to `group_size`.

**Comment by njzjz** [comment]
What is different if you set `group_size` to `total_task_number` // `group_number`?

**Reply by goodluck-xyz** [reply]
If we have 52 pokers and 4 players, `group_size` means dealing cards 13 cards by 13 cards.   `group_number` means deals cards one card by one card.
